newdle/client/src/components/Summary.js

```diff
@@ -0,0 +1,76 @@
+import React, {useState} from 'react';
+import {serializeDate, toMoment} from '../util/date';
+import {useDispatch, useSelector} from 'react-redux';
+import {Button, Container, Icon, Loader} from 'semantic-ui-react';
+import ParticipantTable from './ParticipantTable';
+import {getNewdle} from '../selectors';
+import {updateNewdle} from '../actions';
+import client from '../client';
+import styles from './Summary.module.scss';
+
+export default function Summary() {
+  const [submitting, setSubmitting] = useState(false);
+  const [error, setError] = useState('');
+  const [finalDate, setFinalDate] = useState(null);
+  const newdle = useSelector(getNewdle);
+  const dispatch = useDispatch();
+
+  async function update() {
+    setError('');
+    setSubmitting(true);
+    let updatedNewdle;
+    try {
+      updatedNewdle = await client.setFinalDt(newdle.code, {final_dt: finalDate});
```

Comment:
The second argument should just be `finalDate` now that the method is called `setFinalDt`